content/cli/usages.md

```diff
@@ -70,7 +70,7 @@ $ nest g <schematic> <name> [options]
 | `pipe`        | `pi`  | Generate a pipe declaration.                                                                        |
 | `provider`    | `pr`  | Generate a provider declaration.                                                                    |
 | `resolver`    | `r`   | Generate a resolver declaration.
-| `resource`    | `res` | Generate a new CRUD resource. See the [CRUD (resource) generator](/recipes/crud-generator) for more details.                                                                    |
+| `resource`    | `res` | Generate a new CRUD resource. See the [CRUD (resource) generator](https://docs.nestjs.com/recipes/crud-generator) for more details.                                                                    |
```

Comment:
What's the reason for this change?

Reply:
(this question applies to most of the updates in this PR)

Reply:
I did these PRs, to remove the warnings that occur when we do `npm run build` (see screenshot)

To remove them I added this part `https://docs.nestjs.com`to each url where there was the warning.
It should always be added to have no warnings in the future.

<img width="1026" alt="screenshot" src="https://user-images.githubusercontent.com/11542387/147460827-da9381a2-7b96-497e-b8b4-18a159315102.png">

Reply:
The local version of the application won't be working as expected now as every link will route you to the deployed version of the app (instead of the local one running on your machine) making the development process quite frustrating

Reply:
Got it, can this be fixed when a new version of NestJS (v9) comes out by updating the site?

Reply:
Not sure if I'm following(?). Explicitly adding `docs.nestjs.com` to the link just makes no sense as the application won't run as expected between different environments.

Reply:
Understood , In many URLs on the site this is already there (see screenshot) but in others it is missing and generates the warning.

<img width="1287" alt="screensho2" src="https://user-images.githubusercontent.com/11542387/147463432-7d51d1b3-ada9-4c54-aabf-a1f3f2331cca.png">

Reply:
These are the same warnings that you also find in the Netfily console (see screenshot)

<img width="768" alt="screenshot3" src="https://user-images.githubusercontent.com/11542387/147464101-9eb59e4a-24cc-4f63-9fc7-7df18b5df4f3.png">

Reply:
> Understood, In many URLs on the site this is already there (see screenshot) but in others it is missing and generates the warning.

We use `docs.nestjs.com` explicitly **only** if we want a link to be opened in a new tab

Reply:
Ok, so we leave the warnings like this? 
You can also find them in the Netfily console (you saw the screenshot above)? 

However it would be bad for each link to open a new tab, it is true that it depends on the cases, but for a guide it would be interesting, so you can pick up where you were reading.

Reply:
> Ok, so we leave the warnings like this?

yeah these warnings are acceptable ✅

Reply:
Ok 👍, if you want these links to open in another tab ok, otherwise close these PRs as well. Then if you want to think about it let me know, no problem for me. 😉 👍